ddtrace/tracer/sqlcomment.go

```diff
@@ -31,15 +31,16 @@ const (
 
 // Key names for SQL comment tags.
 const (
-	sqlCommentKeySamplingPriority = "ddsp"
-	sqlCommentTraceID             = "ddtid"
-	sqlCommentSpanID              = "ddsid"
-	sqlCommentService             = "ddsn"
-	sqlCommentDBService           = "dddbs"
-	sqlCommentVersion             = "ddsv"
-	sqlCommentEnv                 = "dde"
+	sqlCommentTraceParent   = "traceparent"
+	sqlCommentParentService = "ddsn"
```

Comment:
Is it too late to align the keys with their names here? e.g. `ParentService` is `ddsn` but could it be `ddps`? If the answer is it's too late to change or it can't change for some reason that's fine too

Reply:
I think it might be our last chance but there's still time. I'll update.

Reply:
I renamed `ddsn` => `ddps` and `ddsv` to `ddpv`. I kept environment as is (dde) because I guess that's not specific to the parent. But I'm not totally convinced. `Env` came up in conversations as being a little bit special. Thoughts?

Reply:
I'm not familiar with any "special-ness" around `env` but `dde` seems fine to me / obvious what it means. I was mostly interested in the change since I know that at some point in the future someone will come along and say "why is Parent Service `ddsn`?" and no one will remember :P This looks good